tests/pthread/test_pthread_file_io.cpp

```diff
@@ -7,6 +7,7 @@
 #include <emscripten.h>
 #include <emscripten/threading.h>
 #include <assert.h>
+#include <stdio.h>
```

Comment:
Revert?

Reply:
That is to enable that file to build outside the test harness. The test harness adds implicit stdio.h include somewhere in the added with_report_result concatenation or something like that - the file won't build locally otherwise.